From 2da1ce15dfc85cfbd390b4cfb9350a13a6b47003 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=D0=A0=D1=83=D1=81=D0=BB=D0=B0=D0=BD=20=D0=98=D0=B6=D0=B1?=
 =?UTF-8?q?=D1=83=D0=BB=D0=B0=D1=82=D0=BE=D0=B2?= <lrn1986@gmail.com>
Date: Mon, 13 Apr 2015 00:29:21 +0000
Subject: [PATCH 01/20] [GTK][W32] Don't include non-existing
 WTF/WTFHeaderDetection.h in 2.4.x

WTF/WTFHeaderDetection.h seems to be from some other universe,
there are no rules to generate it in webkit-2.4, except for MSVS projects.
---
 Source/WTF/wtf/Platform.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Source/WTF/wtf/Platform.h b/Source/WTF/wtf/Platform.h
index d918b43..2ebdeea 100644
--- a/Source/WTF/wtf/Platform.h
+++ b/Source/WTF/wtf/Platform.h
@@ -611,7 +611,9 @@
 #define HAVE_ALIGNED_MALLOC 1
 #define HAVE_ISDEBUGGERPRESENT 1
 
+#ifdef _MSC_VER
 #include <WTF/WTFHeaderDetection.h>
+#endif
 
 #endif
 
-- 
1.8.5.3

